Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -186,13 +202,16 @@ function Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric
         {
             Write-Host "[Step 1 of 4] Validating and installing the modules required to run the script and validating the user..."
```

Comment:
[Step 1 of 4] Validate and install the modules required to run the script and validate the user